extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -74,6 +74,7 @@ function ScrollerWithRef({children, loop, restingIndex, setRestingIndex}, ref) {
     offsetRef,
     pivotIndex,
     restingIndex,
+    styles: useStyles(),
```

Comment:
Ideally a hook should be executed at a higher level and not a part of another expression. This might seem a bit like a nit, but hooks are explicitly top-level commands in a component. For instance, hooks cannot be executed conditionally. E.g. `if (x) {useStyles()}` is not allowed. This might not happen in this case, but I think it's safer to hoist it higher up.